src/ipyaladin/aladin.py

```diff
@@ -0,0 +1,430 @@
+                    "A MOC can be given as an URL, a dictionnary, or a mocpy.MOC "
+                    "object. To read mocpy.MOC objects, you need to install the mocpy "
+                    "library with 'pip install mocpy'."
+                ) from imp
+
+    def add_moc_from_URL(
+        self, moc_URL: str, moc_options: Optional[dict] = None
+    ) -> None:
+        """Load a MOC from a URL and display it in Aladin Lite widget.
+
+        Parameters
+        ----------
+        moc_URL: str
+            An URL to retrieve the MOC from
+        moc_options: dict
+
+        """
+        warnings.warn(
+            "add_moc_from_URL is replaced by add_moc that detects automatically"
+            "that the MOC was given as an URL.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        if moc_options is None:
+            moc_options = {}
+        self.add_moc(moc_URL, **moc_options)
+
+    def add_moc_from_dict(
+        self, moc_dict: dict, moc_options: Optional[dict] = None
+    ) -> None:
+        """Load a MOC from a dict object and display it in Aladin Lite widget.
+
+        Parameters
+        ----------
+        moc_dict: dict
+            It contains the MOC cells. Key are the HEALPix orders, values are the pixel
+            indexes, eg: {"1":[1,2,4], "2":[12,13,14,21,23,25]}
+        moc_options: dict
+
+        """
+        warnings.warn(
+            "add_moc_from_dict is replaced by add_moc that detects automatically"
+            "that the MOC was given as a dictionary.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        if moc_options is None:
+            moc_options = {}
+        self.add_moc(moc_dict, **moc_options)
+
+    def add_table(self, table: Union[QTable, Table], **table_options: any) -> None:
+        """Load a table into the widget.
+
+        Parameters
+        ----------
+        table : astropy.table.table.QTable or astropy.table.table.Table
+                table that must contain coordinates information
+
+        Examples
+        --------
+        Cell 1:
+        >>> from ipyaladin import Aladin
+        >>> from astropy.table import QTable
+        >>> aladin = Aladin(fov=2, target='M1')
+        >>> aladin
+        Cell 2:
+        >>> ra = [83.63451584700, 83.61368056017, 83.58780251600]
+        >>> dec = [22.05652591227, 21.97517807639, 21.99277764451]
+        >>> name = ["Gaia EDR3 3403818589184411648",
+                    "Gaia EDR3 3403817661471500416",
+                    "Gaia EDR3 3403817936349408000",
+                   ]
+        >>> table = QTable([ra, dec, name],
+                            names=("ra", "dec", "name"),
+                            meta={"name": "my sample table"})
+        >>> aladin.add_table(table)
+        And the table should appear in the output of Cell 1!
+
+        """
+        # this library must be installed, and is used in votable operations
+        # http://www.astropy.org/
```

Comment:
I think we can remove the comment as astropy is in our mandatory dependencies now